api/views/phones.py

```diff
@@ -348,6 +364,8 @@ def inbound_sms(request):
         real_phone = RealPhone.objects.get(user=relay_number.user, verified=True)
     except ObjectDoesNotExist:
         raise exceptions.ValidationError("Could not find relay number.")
+    if not relay_number.enabled:
+        return response.Response(status=200, data={"message": "Number is not accepting messages."})
```

Comment:
Is a `200` the right response code?

Reply:
It looks like 404 works as well, but a bigger issue might be that we're returning HTML rather than a JSON response. Twilio error logs are showing "12200 - Schema Validation Warning" even for correct responses. It looks like we're supposed to response with [Twilio Markup Language](https://www.twilio.com/docs/glossary/what-is-twilio-markup-language-twiml)

Reply:
Oh dang, good catch! I worried that a `400` response code might cause Twilio to retry. But it looks like they don't do that? (FXA does, and sometimes AWS does too.)

I'll also fix this to return XML like the `inbound_call` does. Which reminds me - I need to add this `enabled` check to `inbound_call` too!!

Reply:
Okay, I added https://github.com/mozilla/fx-private-relay/pull/2259/commits/7cd0dfd780507e41094e3fbd5c84a6a63fd32921 to return 400's and valid TwiML.